utils/pom.xml

```diff
@@ -137,18 +137,6 @@
                     </execution>
                 </executions>
             </plugin>
-            <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
Why removal of this?
I admit, I am not sure what it does. But it was not added with the file in question.

Reply:
The point was the following:
The custom `NativeIO` was needed in two places in tests - in `utils` and in `spark-jobs`. Instead of adding the same file twice in tests, you'd want to define a dependency from one module to the other. Except normally, the tests are not packaged, only `main ` is - so to reuse the class, you either move it from `test/` to `main/` if it makes sense or you define a test-jar to be built and depend on the test-jar. This was the latter option.

You can see that in `spark-jobs/pom.xml:125`, the util test-jar dependency is also removed.

Reply:
Thank you for you explanation.